configs/transposit.json

```
@@ -7,7 +7,9 @@
     "https://www.transposit.com/docs-sitemap.xml"
   ],
   "sitemap_alternate_links": true,
-  "stop_urls": [],
+  "stop_urls": [
+    "^https:\/\/transposit\.com\/docs\/references\/connectors\/(?:[a-z0-9_]*)"
+  ],
```

```suggestion
  "stop_urls": [
    "/docs/references/connectors/"
  ],
```

Just adding this is enough to exclude `connectors` and his childrens